1-the-basics/form.html

```diff
@@ -0,0 +1,114 @@
+                        <label for="email" id="user-email" >Yor Email <abbr title="required"> *</abbr>
+                          <input id="email"  type="email" aria-required="true" required>
+                        </label>
+                        <label for="phonenumber" id="user-phone-number">Yor phonenumber <abbr title="required"> *</abbr>
+                          <input id="phonenumber"  type="number" aria-required="true" required>
+                        </label>
+                    </fieldset>
+                    
+                    <fieldset>
+                      <legend>Please select one or more disciplines</legend>
+                      <label for="basketball" class="resalt">
+                          <input type="checkbox" id="basketball" value="Basketball"> Basketball
+                      </label>
+                      <br>
+                      <label for="swimming" class="resalt">
+                        <input type="checkbox" id="swimming" value="Swimmingr">Swimming
+                      </label>
+                      <br>
+                      <label for="football" class="resalt">
+                          <input type="checkbox" id="football" value="Football"> Football
+                      </label>
+                      <br>
+                      <label for="volleyball" class="resalt">
+                          <input type="checkbox" id="volleyball" value="Volleyball"> Volleyball
+                      </label>
+                      <br>
+                    </fieldset>
+                    <br>
+                    <fieldset>
+                      <legend>Choose your gender: </legend>
+                      <label for="gender">Gender: </label><select id="gender">
+                          <option value="male" selected required>Male</option>
+                          <option value="female" required>Female</option>
+                          <option value="other" required>Other</option>
+                      </select>
+                    </fieldset>
+                    <fieldset>
+                        <legend>Choose your socioeconomic status:</legend>
+                        <input required id="low" type="radio" name="status" value="Low">
+                        <label for="low">Low</label>
+                        <input required id="medium" type="radio" name="status" value="Hedium">
+                        <label for="medium">Medium</label>
+                        <input required id="high" type="radio" name="status" value="High">
+                        <label for="high">High</label>
+                    </fieldset>
+                    <div class="message">
+                      <label for="message">Please enter a message (optional)</label>
+                      <br>
+                      <textarea name="message" id="message" cols="50" rows="5" placeholder="Put a message"></textarea>
+                    </div>
+                    <div style="display: block;">
```

Comment:
```suggestion
                    <div>
```
Remove inline styles, you now have a CSS file to include them.